### PR TITLE
[Quanta] Add and update platform and device files

### PR DESCRIPTION
**Pull request by roberthong-qct**
#### Why I did it
Add device and platform code for ix7-bwde, ix8a-bwde.
Support platform API 2.0 for all quanta platforms except for ix1b
#### How I did it
Implement code
#### How to verify it
for all quanta platforms except for ix1b:
sfputil show eeprom
show platform fan
show platform psu
show platform syseeprom
show platform temp
#### Which release branch to backport (provide reason below if selected)

- [ ] 201811
- [ ] 201911
- [ ] 202006
- [x] 202012

### CODE REVIEW

**Review by geans-pin — APPROVED** [review]

**Review by geans-pin — DISMISSED** [review]

**Review by fk410167 — APPROVED** [review]
Extra alerts are False. Please ignore them

File: device/quanta/x86_64-quanta_ix7_rglbmc-r0/Quanta-IX7-32X/port_config.ini [hunk 1]
```diff
@@ -1,33 +1,33 @@
...
-Ethernet96      113,114,115,116       hundredGigE25     25       100000
-Ethernet100     117,118,119,120       hundredGigE26     26       100000
-Ethernet104     121,122,123,124       hundredGigE27     27       100000
-Ethernet108     125,126,127,128       hundredGigE28     28       100000
-Ethernet112     81,82,83,84           hundredGigE29     29       100000
-Ethernet116     85,86,87,88           hundredGigE30     30       100000
-Ethernet120     89,90,91,92           hundredGigE31     31       100000
-Ethernet124     97,98,99,100          hundredGigE32     32       100000
+# name          lanes                 alias             index    speed    valid_speeds
+Ethernet0       29,30,31,32           hundredGigE1      1        100000   100000,40000
```
Comment (lguohan) [inline device/quanta/x86_64-quanta_ix7_rglbmc-r0/Quanta-IX7-32X/port_config.ini:2]: what is valid_speeds? i do not remember we have introduced this concept in port_config.ini.

Comment (lguohan) [reply device/quanta/x86_64-quanta_ix7_rglbmc-r0/Quanta-IX7-32X/port_config.ini:2]: also i do not see if there is parsing logic on this column.

**Comment by lguohan** [comment]
image

why are these false alerts?

File: device/quanta/x86_64-quanta_ix7_rglbmc-r0/Quanta-IX7-32X/port_config.ini (hunk 1 above)
Comment (roberthong-qct) [reply device/quanta/x86_64-quanta_ix7_rglbmc-r0/Quanta-IX7-32X/port_config.ini]: These are for Broadcom PDE. I removed them in https://github.com/Azure/sonic-buildimage/pull/6971/commits/04323af41a85dd01553488e7195899eab47ada46

**Comment by roberthong-qct** [comment]
In our fan.py, it’s __init__(self, index, is_psu_fan=False).
So if it’s not psu fan, the second argument is not necessary.
Even though I modified it as Fan(fan_index, False) in [Quanta] Add and update platform and device files by roberthong-qct · Pull Request #6971 · Azure/sonic-buildimage (github.com)<https://github.com/Azure/sonic-buildimage/pull/6971/commits/eaf5db193a4e59858e779f9b6d894524b84df7c3>, there are still 2 alerts:
***@***.***D73AB9.B6268EE0]
These are the results of different implementations of __init__ method of class Fan.

**Comment by sonic-qct** [comment]
"azure-pipelines"

**Review by jleveque — APPROVED** [review]

**Pull request merged**